platforms/avr/drivers/spi_master.h

```diff
@@ -54,6 +60,7 @@ spi_status_t spi_transmit(const uint8_t *data, uint16_t length);
 spi_status_t spi_receive(uint8_t *data, uint16_t length);
 
 void spi_stop(void);
+void spi_stop_extended(pin_t slavePin, spi_cs_select_mode_t cs_mode);
```

Comment:
Maybe `cs_mode` should be stored at the start time, similar to how `slavePin` is stored and then used in `spi_stop()`, so that `spi_stop_extended()` won't be needed?

Reply:
Sold! thanks.